### ISSUE TITLE
Error in preferences page path-selector when there is no default value

### ISSUE THREAD

**Issue by hicom150** [webtorrent/webtorrent-desktop#1652]
**What version of this package are you using?**
Latest master commit https://github.com/webtorrent/webtorrent-desktop/commit/afd3b84b406921ecee049c8aeb0d7dc55bdaa76d

**What operating system, Node.js, and npm version?**
OSX Yosemite (10.10.5), Node.js v8.16.0, npm 6.10.2

**What happened?**
In preferences page when you try to push `CHANGE` button in `Folder to watch` or you try to push  `CHANGE` button in `External player` an error message is throw in debug console and dialog does not open

```
Error: Could not call remote method 'showOpenDialog'. Check that the method signature is correct. Underlying error: Default path must be a string
```

**What did you expect to happen?**
I expect the dialog to open

**Are you willing to submit a pull request to fix this bug?**
Sure! 😎

**Comment by hicom150** [comment]
Should be fixed in #1653 😉

**Comment by feross** [comment]
Fixed by https://github.com/webtorrent/webtorrent-desktop/pull/1498
